cpp/src/arrow/compute/exec/hash_join.h

```diff
@@ -112,6 +113,9 @@ class HashJoinImpl {
   virtual void Abort(TaskScheduler::AbortContinuationImpl pos_abort_callback) = 0;
 
   static Result<std::unique_ptr<HashJoinImpl>> MakeBasic();
+
+ protected:
+  util::tracing::Span span;
```

Comment:
Same here, we generally suffix private/protected members with `_`. (See ["Class Data Members"](https://google.github.io/styleguide/cppguide.html#Variable_Names).)